roxie/ccd/ccdserver.cpp

```diff
@@ -4081,7 +4082,7 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
     ruid_t ruid;
     mutable CriticalSection buffersCrit;
     unsigned processed;
-    cycle_t totalCycles;
+    cycle_t activityStats;
```

Comment:
search and replaced too much?
This isn't an ActivityTimeAccumulator type, and is actually total cycles only.
I think should not have renamed this one.

Reply:
yes, reverting this.